cmd/store/query/query.go

```diff
@@ -107,32 +107,34 @@ func (cmd *Command) Run(args ...string) error {
 
 	// set defaults
 	if start != "" {
-		if t, err := parseTime(start); err != nil {
+		t, err := parseTime(start)
```

Comment:
We can reduce these two blocks down by three lines. 

On line 108 define `var err error`, then on line 110:

```go
if cmd.startTime, err = parseTime(start); err != nil {
    return err
}
```

and also line 120:

```go
if cmd.endTime, err = parseTime(end); err != nil {
    return err
}
```